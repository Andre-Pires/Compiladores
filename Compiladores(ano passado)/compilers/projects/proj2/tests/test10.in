float exp4(int a, int b, float c, float d){
  int e;
  float f;
  if(a<=c || b<=d){
    e=a + b;
    f=a + d;
  }else{
    e=a + (int)c;
    f=b + d;
  }
  if(e<=0)
    e=-e + 1;
  if(f<=0)
    f=-f + 1;
  while((f - e) > 0)
    f=f - e;
  return a + b+ c + d + e + f; 
}
