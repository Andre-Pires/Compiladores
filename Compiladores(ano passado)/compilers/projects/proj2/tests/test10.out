warning: type cast int to float, line 4
warning: type cast int to float, line 6
warning: type cast int to float, line 9
warning: type cast int to float, line 13
warning: type cast int to float, line 14
warning: type cast int to float, line 15
warning: type cast int to float, line 16
warning: type cast int to float, line 17
